docs-sources/includes/_Actors.md

```diff
@@ -9,20 +9,62 @@ is assigned a unique session ID.
 As long as a session ID is used and reused by an actor, it will stay active. After a long period
 of non-activity however, the session will expire and the actor will be "logged out" as it were.
 
-## Auth and Session modules
+## Session module
 
 > `lib/index.js`
 
 ```javascript
 mage.useModules([
-  'auth',
   'session'
 ]);
 ```
 
-Freshly bootstrapped MAGE applications already have the auth and the session module activated and
-configured (including a basic Archivist configuration which will store auth-information to disk
-and session-information in memory).
+Freshly bootstrapped MAGE applications already have the session module activated and configured
+(including a basic Archivist configuration which will store session-information in memory).
+
+## Auth module
+
+> `lib/index.js`
+
+```javascript
+mage.useModules([
+  'auth'
+]);
+```
+
+The `auth` module is not enabled by default since it requires a working archivist topic
+to store user data. To enable it make sure you have working vaults then add the following
+to your application:
```

Comment:
Keep in mind, these code blocks are appearing on the right, not below. You should make sure to generate the doc to see the final render of the content.